WorkspaceServer.Tests/Kernel/CSharpKernelTests.cs

```diff
@@ -143,9 +150,11 @@ public async Task it_returns_the_result_of_a_null_expression()
 
             await kernel.SendAsync(new SubmitCode("null"));
 
-            KernelEvents.OfType<ValueProduced>()
+            KernelEvents.Where(k => k.Value is ValueProduced)
```

Comment:
This change in the assertion makes it harder to follow.

I'm seeing this pattern a bunch and a custom assertion extension might be good so that when it fails due to the event not being present in the collection, we can see what events were present in the test output.